### PR TITLE
FG-113: Table column - custom (editable) story

### PR DESCRIPTION
**Pull request by AndyKIron**

### CODE REVIEW

File: projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts [hunk 1]
```diff
@@ -102,7 +111,19 @@ export abstract class InputInlineBaseComponent implements ControlValueAccessor,
...
+        this.isEditMode$
+            .asObservable()
+            .pipe(
+                takeUntil(this.onDestroy$),
+                filter(val => val)
+            )
+            .subscribe(this.handleClickOutSideListener.bind(this));
+
+        this.inputControl.valueChanges.pipe(takeUntil(this.onDestroy$)).subscribe(val => {
+            if (!!val && this.error) {
```
Comment (chenlevin89) [inline projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts:123]: move it to separate function 
like the above sunscribe

File: projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts [hunk 2]
```diff
@@ -208,4 +234,18 @@ export abstract class InputInlineBaseComponent implements ControlValueAccessor,
...
+        const parentRect = this.elementRef.nativeElement.getBoundingClientRect();
+        return (
+            parentRect.left <= event.clientX &&
+            parentRect.right >= event.clientX &&
+            parentRect.top <= event.clientY &&
+            parentRect.bottom >= event.clientY
+        );
+    }
+
+    private isType(type: InlineInputType): boolean {
```
Comment (chenlevin89) [inline projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts:248]: why ?
why not comapre the enums ?

File: projects/fusion-ui/components/table/v3/stories/custom-cell-edit/custom-cell-edit.component.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import {AfterViewInit, Component, EventEmitter, HostBinding, Input, Output, ViewChild} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {FormControl, FormsModule, ReactiveFormsModule, Validators} from '@angular/forms';
+import {BehaviorSubject, Observable} from 'rxjs';
+import {isNullOrUndefined, isNumber} from '@ironsource/fusion-ui/utils';
+import {InlineInputType, InputInlineComponent} from '@ironsource/fusion-ui/components/input-inline/v3';
+
+@Component({
```
Comment (chenlevin89) [inline projects/fusion-ui/components/table/v3/stories/custom-cell-edit/custom-cell-edit.component.ts:8]: on push straregy !!!!!!!!!!!!!!!!!!!!!!!

File: projects/fusion-ui/components/input-inline/v3/input-inline.component.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => InputInlineComponent),
+            multi: true
+        }
+    ]
+})
+export class InputInlineComponent extends InputInlineBaseComponent implements OnInit {
+    ngOnInit() {
+        super.ngOnInit();
+        this.configByStyle = CONFIG_INPUT_INLINE_BY_UI_STYLE[`style_v3`];
```
Comment (chenlevin89) [inline projects/fusion-ui/components/input-inline/v3/input-inline.component.ts:33]: why do we need it ?? 
this isn't how other component works

File: projects/fusion-ui/components/table/v3/stories/custom-cell-edit/custom-cell-edit.component.ts (hunk 3 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/table/v3/stories/custom-cell-edit/custom-cell-edit.component.ts:8]: Ops... added

File: projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts (hunk 2 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts:248]: You are right it looks weird, but sometimes (in the storybook) I get it as a string like "InlineInputType.Currency". So just added one more check. Will check it again.

File: projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts (hunk 1 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts:123]: Sure.

**Review by chenlevin89 — APPROVED** [review]

**Pull request merged**